internal/providers/terraform/google/logging_billing_account_bucket_config.go

```diff
@@ -0,0 +1,46 @@
+package google
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+func GetLoggingBillingAccountBucketConfigRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "google_logging_billing_account_bucket_config",
+		RFunc: NewLoggingBillingAccountBucket,
+	}
+}
+
+func NewLoggingBillingAccountBucket(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var loggingData *decimal.Decimal
+	if u != nil && u.Get("monthly_logging_data_gb").Exists() {
+		loggingData = decimalPtr(decimal.NewFromInt(u.Get("monthly_logging_data_gb").Int()))
+	}
+
+	costComponents := []*schema.CostComponent{
+		{
+			Name:            "Logging data",
+			Unit:            "GB-months",
```

Comment:
```suggestion
			Unit:            "GB",
```

Since users pay for data that is processed, so it's not a storage (per month) fee